pkg/cmd/cli/install/install.go

```diff
@@ -303,6 +303,15 @@ func (o *InstallOptions) Run(c *cobra.Command, f client.Factory) error {
 		}
 	}
 
+	// Check velero vsphere plugin image repo
+	repo, tag, err := ParsePluginImage(f)
+	if err != nil {
+		fmt.Println("Failed to parse velero vsphere plugin image repository and tag.")
```

Comment:
If we can't parse the tag, let's use the one that was compiled in.

Reply:
InstallOption is initialized with default image, here o.Image will be changed only when tag is successfully parsed.
if err != nil {
		fmt.Println("Failed to parse velero vsphere plugin image repository and tag.")
	} else {
		if repo != "" && tag != "" {
			o.Image = repo + "/" + utils.DataManagerRepository + ":" + tag
		}
	}